modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderContext.tsx

```diff
@@ -33,6 +33,7 @@ const initialState = {
 	elements: [] as Elements<
 		ObjectDefinitionNodeData | ObjectRelationshipEdgeData
 	>,
+	isConnectable: false,
```

Comment:
Hey @igor-franca, I think this name is too generic, in a large project like modal builder, what isConnectable? I think we can rename this to nodeHandleConnectable or something like that, what do you think?

Reply:
Thanks @danvb112! It really is a better name. Done 🤙🏾